specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/iotSecuritySolutionAnalytics.json

```diff
@@ -0,0 +1,711 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/iotSecuritySolutions/{solutionName}/Analytics/current": {
```

Comment:
>current [](start = 147, length = 7)

For a singleton resource, the ARM naming convention is to call it `default`. Can you change the name?

Reply:
The resource type naming convention is camelCasePlural. Can you change the name to `analytics`?

---
In reply to: [295953066](https://github.com/Azure/azure-rest-api-specs/pull/6160#discussion_r295953066) [](ancestors = 295953066)

Reply:
On 2nd thought analytics sounds really vague. I think analyticsModels or analyticModels would fit better and matches the name of the definition.

---
In reply to: [295954442](https://github.com/Azure/azure-rest-api-specs/pull/6160#discussion_r295954442) [](ancestors = 295954442,295953066)

Reply:
I understood that when I have only one model data I need to use current.
I have changed it to default